.readthedocs.yaml

```diff
@@ -0,0 +1,13 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+sphinx:
+  configuration: docs/source/conf.py
+
+python:
+  install:
+  - requirements: requirements-docs.txt
```

Comment:
There is a pending PR that will refactor this path: https://github.com/scrapinghub/spidermon/pull/407